platform/linuxbsd/os_linuxbsd.cpp

```diff
@@ -211,14 +211,15 @@ String OS_LinuxBSD::get_name() const {
 
 String OS_LinuxBSD::get_systemd_os_release_info_value(const String &key) const {
 	static String info;
-	if (info.is_empty()) {
```

Comment:
If you remove this, you defeat the purpose of `info` being static as it will be recomputed each time.